test/e2e/infraprovider/providers/kind/ovs.go

```diff
@@ -0,0 +1,93 @@
+package kind
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/ovn-org/ovn-kubernetes/test/e2e/deploymentconfig"
+
+	e2epodoutput "k8s.io/kubernetes/test/e2e/framework/pod/output"
+)
+
+const (
+	secondaryBridge = "ovsbr1"
+)
+
+func addOVSBridge(podNamespace, podName string, bridgeName string) error {
```

Comment:
hit; its more like ensureOVSBridge

Reply:
i think youll always create this bridge and wont rely on it created for you right? so i think you should fail if its already there because we may have a test problem we want to highlight then below we can safely remove this bridge. wdyt?

Reply:
itll stop us from removing someones bridge by mistake when we call deleteOVSBridge.

Reply:
The problem with that is we have one suite and its not ideal to setup something when the test may not even use it- i think, personally, good practise is for the tests to create the resources it needs at test run time. If thats not possible, then fall back to precreated.